docs/plugins/artifactory.md

```diff
@@ -13,12 +13,13 @@ values_yaml: |
         artifactory:
           service:
             type: NodePort
-            nodePort: 30002
+          nodePort: 30002
```

Comment:
<img width="778" alt="image" src="https://user-images.githubusercontent.com/36830265/183014528-dc2372bd-f95c-4946-84ae-bbbf51c96c57.png">
You should use the same indentation

Reply:
https://github.com/jfrog/charts/blob/master/stable/artifactory/values.yaml

Reply:
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/36830265/183015114-5d469fd0-e17d-4092-95d6-91ed5048e6fe.png">